### PR TITLE
Get NZB release name for failed.db using guessit

### PR DESCRIPTION
**Pull request by fernandog**
Uses guessit to parse release name

### CODE REVIEW

**Comment by fernandog** [comment]
More 3 days without review and I will merge it

**Comment by p0psicles** [comment]
Can you explain? I thought we used guessit for this?

**Comment by fernandog** [comment]
@p0psicles you are right. i rewrote to use guessit

**Comment by fernandog** [comment]
@OmgImAlexis can you also test this please as you use NZB?

**Comment by fernandog** [comment]
@medariox don't forget this one after the other PR please

**Comment by fernandog** [comment]
@medariox please?

**Comment by fernandog** [comment]
still waiting....

File: medusa/failed_processor.py [hunk 1]
```diff
@@ -36,39 +39,41 @@ def __init__(self, dirName, nzbName):
...
         self._log(u'Failed download detected: ({nzb}, {dir})'.format(nzb=self.nzb_name, dir=self.dir_name))
 
-        releaseName = show_name_helpers.determineReleaseName(self.dir_name, self.nzb_name)
-        if not releaseName:
-            self._log(u'Warning: unable to find a valid release name.', logger.WARNING)
-            raise FailedPostProcessingFailedException()
-
         try:
-            parsed = NameParser().parse(releaseName)
+            parsed = NameParser().parse(self.nzb_name or self.dir_name)
```
Comment (fernandog) [inline medusa/failed_processor.py]: @medariox this is what Im concern

Dir name is NOT the nzb name right? What can I add here that will always be the nzb folder name?

**Pull request closed without merging**